src/plot_utils.py

```diff
@@ -79,7 +79,7 @@ def mi_hist(bdb, generator, col1, col2, num_samples=1000, bins=5):
 
 
 def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
-        col_ordering=None):
+            col_ordering=None):
```

Comment:
We have not been using all of pylint style (because many of us disagree with its suggestions, see for example the Venture [pylintrc](https://github.com/probcomp/Venturecxx/blob/master/tool/pylintrc)) and this is one instance where I think pylint's suggestion makes the code less readable.